html/apps/signup/css/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Lato Regular'), local('Lato-Regular'), url(http://themes.googleusercontent.com/static/fonts/lato/v7/qIIYRU-oROkIk8vfvxw6QvesZW2xOQ-xsNqO47m55DA.woff) format('woff');
+}
+
+body {
+	background: #448ed3 ;
+	font-family: "Lato" ;
+}
+img{
+    width:15%;
+    height: 15%
+    max-width:30px;
+    display: block;
+    margin: 0 auto;
+}
+p {
+	font-family: "Lato";
+	font-size: 35px;
+	color: #fff;
+}
+.form {
+	width:375px;
+	height: auto;
+	margin: auto;
+}
+
+.form input {
+	border: none;
+	background: #fff;
+  	font-family:Lato ;
+  	font-weight:700 ;
+	display: block;
+	height: 45px;
+	font-size: 15px;
+	outline: none;
+	width: calc(100% - 24px) ;
+	margin: auto;
+	padding: 6px 12px 6px 12px;
+}
+.bar {
+	width: 100%;
+	height: 1px;
+	background: #fff ;
+}
+.bar i {
+	width: 95%;
+	margin: auto;
+	height: 1px ;
+	display: block;
+	background: #d1d1d1;
+}
+.form input[type="text"] {
+	border-radius: 7px 7px 0px 0px ;
+}
+.form input[name = "pass"] {
+	border-radius: 0px 0px 0px 0px ;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l59)